fix appveyor builds

bacongobbler opened the pull request:
Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

Commit:
fix appveyor builds

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

adamreese commented on cmd/helm/helm_test.go:
```diff
@@ -33,28 +33,15 @@ import (
 
 func testTempDir(t *testing.T) string {
 	t.Helper()
-	d, err := ioutil.TempDir(testingDir, "helm")
```
!!!This is important for cleanup!!!

Commit:
test from a non-root directory

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

Commit:
fix issue on windows with using testingDir as input

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

Commit:
fix golden file

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

Commit:
wrap filepaths in single quotes

mattn/go-shellwords (used only in the root tests) doesn't like
windows-style filepaths as arguments. Wrapping it in single quotes
prevents go-shellwords from stripping all the forward slashes as escape
codes.

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

Commit:
remove stray fmt.Println statements; add note on `in()`

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>
